scripts/flutter.sh

```
@@ -51,47 +51,47 @@ setup_flutter() {
 
     # flutter コマンド存在確認 (fvm管理下のパスで)
-    if ! command_exists flutter; then
+    if ! command -v flutter; then
         handle_error "Flutter コマンド (fvm管理下) が見つかりません"
         return 1
     fi
 
     # Flutterのパスを確認 (fvm管理下のパス)
     FLUTTER_PATH=$(which flutter)
-    log_info "Flutter PATH (fvm): $FLUTTER_PATH"
+    echo "[INFO] "Flutter PATH (fvm): $FLUTTER_PATH""
```

Shellcheck (0.10.0)

[error] '(' is invalid here. Did you forget to escape it?

(SC1036)